src/components/Card/Card.module.css

```diff
@@ -40,15 +40,15 @@
  * 1) Change the orientation to display header, body, and footer in a row
  */
 .card--horizontal {
-  flex-direction: row;
+  flex-direction: row; /* 1 */
 }
 
 /**
  * Centered card
  * 1) Card where the content is centered
```

Comment:
I think this comment can be removed since the styling is self-explanatory here as well